src/api/spec/bootstrap/features/webui/users/users_notifications_settings_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'browser_helper'
+
+RSpec.feature "User's notifications settings", type: :feature, js: true do
+  let(:user_with_groups) { create(:user_with_groups, login: 'moi_wg') }
+
+  scenario 'when a user is in some group' do
+    login user_with_groups
+    visit user_notifications_path
+    group_title = user_with_groups.groups.first.title
+
+    expect(page).to have_content 'You will receive emails from the checked groups'
+    # Using `visible: :all` for the Bootstrap version (as the checkboxes aren't found otherwise)
+    expect(page).to have_checked_field(group_title, visible: :all)
+    uncheck(group_title, allow_label_click: true)
+    expect(page).to have_content 'Notifications settings updated'
+    expect(page).to have_content 'You will receive emails from the checked groups'
+    # Using `visible: :all` for the Bootstrap version (as the checkboxes aren't found otherwise)
+    expect(page).to have_unchecked_field(group_title, visible: :all)
+  end
+end
```

Comment:
Are you also testing somewhere that the right things happen apart from the checkbox status?

Reply:
> Are you also testing somewhere that the right things happen apart from the checkbox status?

No. This test is copied from the [spec/features/webui/users/users_notifications_settings_spec.rb
](https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/features/webui/users/users_notifications_settings_spec.rb) one and adapted to work with bootstrap. In case we should check for the right things happen apart from the checkbox status both tests should be modified.

Reply:
A test not testing what really happens makes little sense. At least `expect` that the data changes in the way you want it to...

Reply:
In the old feature test clicking the 'Update' button was causing a page reload. Since this is using ajax now, you would have to manually reload the page (visit user_notifications_path) to be sure that the new state is stored in the server

Reply:
You can also just inspect the data...

Reply:
Sure, but it has it's charm test this via the user interface given that it's a feature test.